src/components/CreateCoursePage/CreateCourseForm.jsx

```diff
@@ -121,32 +134,64 @@ class BaseCreateCourseForm extends React.Component {
             }
             required
           />
-          <Field
-            name="enrollmentTrack"
-            component={RenderSelectField}
-            options={this.getEnrollmentTrackOptions()}
-            label={
-              <FieldLabel
-                id="enrollment-track-label"
-                text="Enrollment track"
+          {usingCourseType ? (
+            <React.Fragment>
+              <Field
+                name="type"
+                component={RenderSelectField}
+                options={courseTypeOptions}
+                label={
+                  <FieldLabel
+                    id="course-type-label"
+                    text="Course Type"
+                    required
+                    helpText={(<div><p>TODO: Come up with type helpText</p></div>)}
```

Comment:
Ahem. (Same TODO on runs too)

Reply:
Leaving these as TODO's until we can demo/show this to Product and come up with useful help text together.